core/audits/accessibility/html-xml-lang-mismatch.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license Copyright 2023 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/**
+ * @fileoverview Ensures that, if present, the `[xml:lang]` attribute value in an HTML document has
+ * the same value as the `[lang]` attribute.
+ * See base class in axe-audit.js for audit() implementation.
+ */
+
+import AxeAudit from './axe-audit.js';
+import * as i18n from '../../lib/i18n/i18n.js';
+
+const UIStrings = {
+  /** Title of an accesibility audit that evaluates if the xml:lang attribute, if present, has the same value as the `lang` attribute. This title is descriptive of the successful state and is shown to users when no user action is required. */
+  title: '`<html>` element has an `[xml:lang]` attribute with the same value as the `[lang]` ' +
+      'attribute.',
```

Comment:
The [axe docs](https://dequeuniversity.com/rules/axe/4.7/html-xml-lang-mismatch) indicate that `lang` and `xml:lang` can have the same base language, although [it isn't super clear](https://dequeuniversity.com/rules/axe/4.7/html-xml-lang-mismatch#:~:text=If%20required%2C%20duplicate%20the%20value%20of%20the%20lang%20attribute%20as%20the%20value%20of%20the%20xml%3Alang%20attribute%20on%20the%20html%20element.). I was able to verify that the same base language does pass this axe check.

```suggestion
  title: '`<html>` element has an `[xml:lang]` attribute with the same base language as the `[lang]` ' +
      'attribute.',
```